ViewModel/ImpactAreaScenario/Results/PerformanceAssuranceOfThresholdVM.cs

```diff
@@ -1,55 +1,45 @@
 ﻿using metrics;
 using System.Collections.Generic;
 using HEC.FDA.ViewModel.ImpactAreaScenario.Results.RowItems;
+using System;
 
 namespace HEC.FDA.ViewModel.ImpactAreaScenario.Results
 {
     public class PerformanceAssuranceOfThresholdVM :  PerformanceVMBase
     {
 
-        public PerformanceAssuranceOfThresholdVM(List<ThresholdComboItem> metrics)
+        public PerformanceAssuranceOfThresholdVM(metrics.Results iasResult, List<ThresholdComboItem> metrics)
         {
-            loadDummyData(metrics);
+            LoadData(iasResult, metrics);
         }
 
-
-        private void loadDummyData(List<ThresholdComboItem> metrics)
+        private void LoadData(metrics.Results iasResult, List<ThresholdComboItem> metrics)
```

Comment:
Rename your metrics variable to something else and remove the metrics namespace from results.